library/compiler-builtins/builtins-shim/Cargo.toml

```diff
@@ -17,7 +17,6 @@ repository = "https://github.com/rust-lang/compiler-builtins"
 license = "MIT AND Apache-2.0 WITH LLVM-exception AND (MIT OR Apache-2.0)"
 edition = "2024"
 publish = false
-links = "compiler-rt"
```

Comment:
I think this indicates a problem and I'd like us to investigate, as you mentioned, it'sa concerning and I want to be sure of the fix here...